src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -51,6 +53,14 @@
 LOCAL_STORAGE_BATCH_SIZE = 1
 
 
+class CustomStrongTypeConnectionConfigs:
+    PREFIX = "promptflow.custom.connection."
+    TYPE = "custom_type"
+    MODULE = "module"
+    FULL_TYPE = PREFIX + TYPE
```

Comment:
FULL_TYPE_KEY ? FULL_TYPE is a little confuse at first glance.

Reply:
What does `FULL` mean here?

Reply:
updated to PROMPTFLOW_TYPE_KEY and PROMPTFLOW_MODULE_KEY